csharp/ql/src/Bad Practices/Implementation Hiding/AbstractToConcreteCollection.qhelp

```diff
@@ -18,7 +18,7 @@ more difficult to change which implementation you are using at a later date.</p>
 </example>
 <references>
 
-  <li>C# Corner, <a href="http://www.c-sharpcorner.com/UploadFile/rmcochran/csharp_interrfaces03052006095933AM/csharp_interrfaces.aspx">C# Interface Based Development</a>.</li>
+  <li>C# Corner, <a href="https://www.c-sharpcorner.com/article/C-Sharp-interface-based-development/">C# Interface Based Development</a>.</li>
```

Comment:
This is just me happily following a redirect which coincidentally fixes the typo check-spelling complained about. As a bonus, it changes an `http` url to an `https` url.